### PR TITLE
Fix Travis' config

### PR DESCRIPTION
**Pull request by ChristophWurst**
* Remove duplicate `matrix` entry
* Move `sudo` to the top
* Separate passing and failing code checker steps
* Simplify build matrix
* Fix indentation

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
+sudo: true
 language: php
-
 php:
   - 7
```
Comment (rullzer) [inline .travis.yml]: Ah we'd also need 5.6, 7.1 and 7.2 then :wink:

Comment (ChristophWurst) [reply .travis.yml]: Done. Obviously the build matrix grew but the test scripts are fast, so no big deal IMO

**Comment by rullzer** [comment]
- [ ] Seems to still miss a 5.6 linter
- [ ] Should not test against stable12 (not supported)
- [x] Should test against stable13
- [ ] Should test against stable14

**Comment by ChristophWurst** [comment]
Fixed: https://travis-ci.org/nextcloud/richdocuments/builds/452262372

**Comment by rullzer** [comment]
Nice thnx a lot!

**Pull request merged**